src/app/public/modules/checkbox/checkbox-label.component.ts

```diff
@@ -1,5 +1,8 @@
 import { Component } from '@angular/core';
 
+/**
+ * Specifies a label for the checkbox.
```

Comment:
Seems like we should ask a developer why we have `SkyCheckboxLabelComponent` and also have a `label` property on `SkyCheckboxComponent`. Which is the preferred method? And why? And what happens if both are used? We might be able to test this in Stackblitz. It may not be worth worrying about since we are hurrying through the conversion and can circle back to this, but it'll be easy to forget about it if we don't address it now.